### ISSUE TITLE
Settings for "Information Fields" not saved

### ISSUE THREAD

**Issue by darealshinji** [mate-desktop/mate-system-monitor#242]
#### Expected behaviour
Should save my settings for the selected Information Fields

#### Actual behaviour
Doesn't save these settings

#### Steps to reproduce the behaviour
Go to settings, disable i.e. "% CPU", close app and open app again, then the "% CPU" field will be activated again.

#### MATE general version
MATE 1.26.0

#### Package version
1.26.0-1

#### Linux Distribution
Ubuntu MATE 22.04 LTS x86_64

**Comment by Kunutza** [comment]
I have the exact same issue

**Comment by vjayer** [comment]
Same here. In the meantime for a workaround, is there a config file we can modify to change the defaults?

**Comment by rbuj** [comment]
In the quick test I performed, everything works correctly: After disabling "% CPU" column in "System Monitor Preferences" dialog:
```
$ dconf dump /org/mate/system-monitor/proctree/
[/]
col-8-visible=false
```
col-8-visible is the 9th element in titles array, zero-based numbering:
https://github.com/mate-desktop/mate-system-monitor/blob/2b767be06f5b5b602e56740275784bb9bf3474c3/src/proctable.cpp#L261

**Comment by mbkma** [comment]
@rbuj  are you on master or on 1.26.0? Because I just found that this PR https://github.com/mate-desktop/mate-system-monitor/pull/224 might fix this and should be in a new release 1.26.1.

**Comment by rbuj** [comment]
I tried it on version 1.26.0, using the fedora 36 package.

**Comment by rbuj** [comment]
I reproduced the issue using `--disable-wnck`, and #224 fixed it, since the index COL_MEMXSERVER must be avoided.
```
CFLAGS="-Wconversion -Wunused-macros -Wunused-parameter" CXX=clang++ CC=clang ./autogen.sh --prefix=/usr --enable-debug --enable-compile-warnings=maximum --disable-wnck && make &> make.log && sudo make install
```

**Comment by mbkma** [comment]
ok, then #224  should be included in a new 1.26.1 release.
